.github/CHANGELOG.md

```diff
@@ -53,7 +60,7 @@
   weight_shapes = {"weights_0": 3, "weight_1": 1}
   qlayer = qml.qnn.TorchLayer(qnode, weight_shapes)
   ```
-
+  
```

Comment:
Lol I tried to modify this using GitHub's 'suggested changes' option on the PR a while back, and gave up --- GitHub struggled to deal with a suggested change, when the suggested change involved manipulating three backticks 😆